app/workers/rapid_ready_for_decision/rrd_processor.rb

```diff
@@ -93,18 +76,36 @@ def patient_info
     end
 
     def icn
-      account_record = account
-      raise AccountNotFoundError, "for user_uuid: #{form526_submission.user_uuid} or their edipi" unless account_record
+      account_records = accounts
+      if account_records.blank?
+        raise AccountNotFoundError, "for user_uuid: #{form526_submission.user_uuid} or their edipi"
+      end
+
+      return account_records.first.icn.presence if account_records.size == 1
+
+      icns = account_records.pluck(:icn).uniq.compact
+      # Multiple Account records should have the same ICN
+      if icns.size > 1
+        message = "Multiple ICNs found for the user '#{form526_submission.user_uuid}': #{icns}"
+        form526_submission.send_rrd_alert_email('RRD Multiple ICNs found warning', message)
+      end
 
-      account_record.icn.presence
+      icns.first
     end
 
-    def account
+    def accounts
       account = Account.lookup_by_user_uuid(form526_submission.user_uuid)
-      return account if account
+      return [account] if account
```

Comment:
I wonder if the ID.me or Login.gov find_by's generally return the same account or if they ever return different accounts.

Reply:
They'll only ever return one Account since those fields must be unique: https://github.com/department-of-veterans-affairs/vets-api/blob/fd7236cc981a4fc6401d61800d250a6385be81d1/db/schema.rb#L51-L54